Ginger/Ginger/MainWindow.xaml.cs

```diff
@@ -526,11 +526,12 @@ private void ALMFieldsConfiguration_Click(object sender, RoutedEventArgs e)
 
         private void ALMDefectsProfiles_Click(object sender, RoutedEventArgs e)
         {
-            if(!ALMIntegration.Instance.AlmConfigurations.UseRest && ALMIntegration.Instance.GetALMType() != ALMIntegration.eALMType.Jira)
-            {
-                Reporter.ToUser(eUserMsgKey.ALMDefectsUserInOtaAPI, "");
-                return;
-            }
+            //open defect profile page for each almtype
+            //if(!ALMIntegration.Instance.AlmConfigurations.UseRest && ALMIntegration.Instance.GetALMType() != ALMIntegration.eALMType.Jira)
```

Comment:
i think this check is here because for QC defect opening is supported only with rest API. So the check is still needed but condition needs to be improved